### PR TITLE
BZ#1826605 remove outdated Tech Preview notices for NVDIMM, WebSocket Proxy, Failover, VDSM hook nested, open V Switch OVS,Cinderlib

### PR DESCRIPTION
**Pull request by emarcusRH**
Fixes issue # | \[BZ#1826605](https://bugzilla.redhat.com/show_bug.cgi?id=1826605)  

Changes proposed in this pull request:

- remove tech preview notices in document body and rel notes for:
 NVDIMM, WebSocket Proxy, Failover, VDSM hook nested, open V Switch OVS,Cinderlib
^^^
@emarcusRH Can you update this list? 

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): (please @emarcusRH )

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc [hunk 1]
```diff
@@ -56,13 +56,6 @@ Set up Cinderlib integration
 (Currently in tech preview)
```
Comment (apinnick) [inline source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc:56]: This line has probably been removed from the code. It should be checked.

File: source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc [hunk 2]
```diff
@@ -37,20 +37,17 @@ All hosts in the cluster must use IPv4 or IPv6 for RHV networks, not simultaneou
...
-|Open vSwitch (OVS) cluster type support |Adds Open vSwitch networking capabilities.
 |Shared and local storage in the same data center |Allows the creation of single-brick Gluster volumes to enable local storage to be used as a storage domain in shared data centers.
-|Cinderlib Integration |Leverage CinderLib library to use Cinder-supported storage drivers in {virt-product-fullname} without a Full Cinder-OpenStack deployment. Adds support for Ceph storage along with Fibre Channel and iSCSI storage. The Cinder volume has multipath support on the {hypervisor-fullname}.
 |SSO with OpenID Connect |Adds support for external OpenID Connect authentication using Keycloak in both the user interface and with the REST API.
 |oVirt Engine Backup |Adds support to back up and restore {virt-product-fullname} {engine-name} with the Ansible `ovirt-engine-backup` role.
-|Failover vNIC profile | This feature allows users to migrate a virtual machine connected via SR-IOV with minimal downtime by using a failover network that is activated during migration.
+|Q35 Chipset with UEFI |Virtual machines can use the Q35 chipset and UEFI, with or without SecureBoot.
```
Comment (apinnick) [inline source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc:50]: Did you mean to add Bochs and UEFI? You removed them in https://github.com/oVirt/ovirt-site/pull/2896. Rebase your branch against master. That should take care of it.

**Review by apinnick — CHANGES_REQUESTED** [review]

File: source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc (hunk 1 above)
Comment (emarcusRH) [reply source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc:56]: right! 
need to verify with Dev...

**Comment by michalskrivanek** [comment]
I'm not sure I fully understand the scope of the change - does the removal of tech preview notice for e.g. cinderlib removes the Tech Preview note from everywhere? Or is it just a cleanup of some duplicities? Some of the features in the list need to remain tech preview. Most of them, actually.

**Comment by apinnick** [comment]
> 
> @michal - I was told that these are no longer in Tech Preview: Q35 with UEFI, Boch Emulator, NDIMM, Websocket Proxy, Failover vNIC, VDSM hook for nested virtualization, Open vSwitch (OVS), and Cinderlib https://bugzilla.redhat.com/show_bug.cgi?id=1826605 https://bugzilla.redhat.com/show_bug.cgi?id=2081421

@emarcusRH  
I think there might have been a misunderstanding. The bug is to remove **obsolete** TP notices. However, all the modules listed in the comments need to be reviewed to see which TPs can be removed.

@michalskrivanek 
Can you review this PR and see whether there are TP notes that should NOT be removed? Thanks.

**Comment by mwperina** [comment]
OpenVSwitch Cluster type is Tech Preview.

Configuring Virtual Machines with SR-IOV-Enabled vNICs with minimal downtime was delivered as tech preview in RHV 4.4.6:
https://bugzilla.redhat.com/show_bug.cgi?id=1688177
https://bugzilla.redhat.com/show_bug.cgi?id=1848983
But it's fully supported from RHV 4.4.7:
https://bugzilla.redhat.com/show_bug.cgi?id=1964261

File: source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc [hunk 3]
```diff
@@ -60,10 +60,8 @@ Set up Cinderlib integration
 [IMPORTANT]
 ====
 Cinderlib is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend to use them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-
 For more information on Red Hat Technology Preview features support scope, see link:https://access.redhat.com/support/offerings/techpreview/[Red Hat Technology Preview Features Support Scope].
 ====
-
```
Comment (apinnick) [inline source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc:66]: This blank line should be kept.

File: source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc [hunk 4]
```diff
@@ -46,11 +46,13 @@ For details about IPv6 support, see link:{URL_virt_product_docs}{URL_format}admi
 * `virt-v2v` cannot change the default kernel in the GRUB2 configuration. The kernel configured on the guest operating system is not changed during the conversion, even if a more optimal version is available.
 
 * After converting a Debian or Ubuntu virtual machine from VMware to KVM, the name of the virtual machine's network interface may change, and will need to be configured manually
+
 |NVDIMM host devices |Support for attaching an emulated NVDIMM to virtual machines that are backed by NVDIMM on the host machine. For details, see link:{URL_virt_product_docs}{URL_format}virtual_machine_management_guide/index#conc-nvdimm-host-devices_vm_guide_administrative_tasks[NVDIMM Host Devices].
```
Comment (apinnick) [inline source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc:50]: URL does not resolve to NVDIMM location.

File: source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc [hunk 5]
```diff
@@ -46,11 +46,13 @@ For details about IPv6 support, see link:{URL_virt_product_docs}{URL_format}admi
...
 |Open vSwitch (OVS) cluster type support |Adds Open vSwitch networking capabilities.
 |Shared and local storage in the same data center |Allows the creation of single-brick Gluster volumes to enable local storage to be used as a storage domain in shared data centers.
 |Cinderlib Integration |Leverage CinderLib library to use Cinder-supported storage drivers in {virt-product-fullname} without a Full Cinder-OpenStack deployment. Adds support for Ceph storage along with Fibre Channel and iSCSI storage. The Cinder volume has multipath support on the {hypervisor-fullname}.
 |SSO with OpenID Connect |Adds support for external OpenID Connect authentication using Keycloak in both the user interface and with the REST API.
 |oVirt Engine Backup |Adds support to back up and restore {virt-product-fullname} {engine-name} with the Ansible `ovirt-engine-backup` role.
 |Failover vNIC profile | This feature allows users to migrate a virtual machine connected via SR-IOV with minimal downtime by using a failover network that is activated during migration.
```
Comment (apinnick) [inline source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc:56]: Remove space before "This feature". Actually, maybe you should change it to "Allows users to migrate ..." for consistency.

File: source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc [hunk 6]
```diff
@@ -46,11 +46,13 @@ For details about IPv6 support, see link:{URL_virt_product_docs}{URL_format}admi
...
 
 * After converting a Debian or Ubuntu virtual machine from VMware to KVM, the name of the virtual machine's network interface may change, and will need to be configured manually
+
 |NVDIMM host devices |Support for attaching an emulated NVDIMM to virtual machines that are backed by NVDIMM on the host machine. For details, see link:{URL_virt_product_docs}{URL_format}virtual_machine_management_guide/index#conc-nvdimm-host-devices_vm_guide_administrative_tasks[NVDIMM Host Devices].
 |Open vSwitch (OVS) cluster type support |Adds Open vSwitch networking capabilities.
 |Shared and local storage in the same data center |Allows the creation of single-brick Gluster volumes to enable local storage to be used as a storage domain in shared data centers.
 |Cinderlib Integration |Leverage CinderLib library to use Cinder-supported storage drivers in {virt-product-fullname} without a Full Cinder-OpenStack deployment. Adds support for Ceph storage along with Fibre Channel and iSCSI storage. The Cinder volume has multipath support on the {hypervisor-fullname}.
```
Comment (apinnick) [inline source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc:53]: Cinderlib Integration - "Integration should be lower case.
Change "Leverage" to "Leverages" for consistency.

File: source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc (hunk 4 above)
Comment (apinnick) [inline source/documentation/doc-Release_Notes/topics/ref-Technology_Preview_Features_RHV.adoc:50]: URL does not resolve to NVDIMM host devices section.

File: source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc (hunk 3 above)
Comment (apinnick) [inline source/documentation/common/install/proc-Configuring_the_Red_Hat_Virtualization_Manager.adoc:66]: This blank line should be kept.

**Review by apinnick — CHANGES_REQUESTED** [review]

**Comment by apinnick** [comment]
LGTM

**Pull request merged**